steps/versionbox-deploy/0.0.1/step.yml

```diff
@@ -0,0 +1,56 @@
+title: versionbox-deploy
```

Comment:
This title will be visible on the website, please use more "human readable" title, like: https://github.com/bitrise-io/steps-xcode-archive/blob/master/step.yml#L1